index.js

```diff
@@ -60,13 +60,69 @@ animalToRemove.remove()
 
 /***** Deliverable 1 *****/
 function toggleColor(element) {
-    if (element.style.color === "green") {
+    if (element.style.color === "red") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
+    console.log(element)
 }
 
+
+// header.addEventListener('click', function(event){
+//     toggleColor(event.target)
+// })
+// IF header has children, use header > event.target
+
+header.addEventListener('click', function(){
+    toggleColor(header)
+})
+// console.log('click')
+
+
+
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector('button.like-button')
+
+likeButton.addEventListener('click', function(){
+
+    likes.textContent = `${traveler.likes++} Likes`
+
+    // const likesPtag = document.querySelector('p.likes')
+    // const likesPtag = button.previousElementSibling
+
+})
```

Comment:
First, you find the like button element on the page.Then, you add an event listener to the like button passing in the event type and callback function as arguments. Great work!